packages/picasso/src/Typography/styles.ts

```diff
@@ -44,18 +44,24 @@ PicassoProvider.override(({ palette, typography }) => ({
 export default ({ palette, typography }: Theme) =>
   // All the body variants are mapped to the same MUI variant (body1) -> declaring styles via custom class names
   createStyles({
-    bodyXsmall: {
+    bodyXxsmall: {
```

Comment:
nit: I'm not sure, but it looks like this will be nicer:
```suggestion
    bodyXXsmall: {
```

Reply:
I agree its more readable, but these classnames are composed from props `variant` and `size` in camel case - and correct size type is `xxsmall`